decidim-core/app/views/decidim/messaging/conversations/_add_conversation_users.html.erb

```diff
@@ -6,17 +6,18 @@
       </div>
     </div>
     <div class="row">
-      <div class="row columns mentions__container">
-        <label for="add_conversation_users"><%= t(".add_max_users") %></label>
-        <input id="add_conversation_users" type="text"
-            class="js-multiple-mentions mentions__container"
+      <div class="row columns autocomplete_search" data-name="recipient_id[]" data-multiple="true">
+        <label for="autocomplete"><%= t(".add_max_users") %></label>
+        <input id="autocomplete" type="text"
```

Comment:
Is there some reason why we need to change the ID here?

I think the old ID is more descriptive for the field in question and allows adding possible other autocompletes in the same view.